src/amo/components/Link.js

```diff
@@ -19,7 +19,7 @@ export class LinkBase extends React.Component {
     prependClientApp: PropTypes.bool,
     prependLang: PropTypes.bool,
     to: PropTypes.oneOfType([PropTypes.object, PropTypes.string]),
-    dispatch: PropTypes.func.isRequired,
+    dispatch: PropTypes.func,
```

Comment:
What is dispatch used for in this component? I guess it's in the destructor only to make sure it gets excluded from `customProps`?

Reply:
Yes, this is needed here because I want to exclude it from the list passed to `a`.

Reply:
How does `Link` get the dispatch property to begin with? I don't see a mapper for it.

Reply:
I think it is because of the spread everywhere 😞

Reply:
Going to remove actually. It does not fix the real issue.